### ISSUE TITLE
add functionality to register an instance as a singleton

### ISSUE THREAD

**Issue by doronguttman** [vuevert/Vert-Core#29]
I have a use case were I receive an instance of an object and I would like to make that object available to be injected.
I was not able to use the existing `.addSingleton` as well as do not have access to the global injector.

Ideally, I would like to have something like this:
```typescript
function init(instanceOfExternalType: ExternalType) {
  const app = new App({
    element: "#app",
    RootComponent: Root
  });

  App.addSingletonInstance(ExternalType, instanceOfExternalType);

  app.start();
}
```

**Comment by LancerComet** [comment]
Thank you, I will check it out very soon! 👍

**Comment by LancerComet** [comment]
Merged! Have a try for 1.2.0! 🍺🐸
